src/docfx/build/moniker/MonikersProvider.cs

```diff
@@ -33,6 +33,18 @@ public MonikersProvider(Docset docset)
             Comparer = new MonikerComparer(monikerDefinition);
         }
 
+        public (List<Error> errors, List<string> monikers) GetFileLevelMonikers(Document file, MetadataProvider metadataProvider)
```

Comment:
>GetFileLevelMonikers [](start = 59, length = 20)

Can we consolidate these two into one method?
The usage of the one below seems the same as this one.

Reply:
as we talked offline, the other parts' file metadata is used in other places, so two methods are ok.

Reply:
we still can consolidate with inputting `metadata`, right?